### PR TITLE
Disable leader election in Helm charts

### PR DESCRIPTION
**Pull request by AkshaySainiDell**
#### Is this a new chart?

Yes

#### Which issue(s) is this PR associated with:

- https://github.com/dell/csm/issues/1106
- https://github.com/dell/csm/issues/1099
- https://github.com/dell/csm/issues/1086
- https://github.com/dell/csm/issues/1110

#### Checklist:

[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]

- [x] Chart Version bumped
- [ ] Variables are documented in the chart README.md
- [ ] Title of the PR starts with the chart name (e.g. `[charts_dir/mychartname]`) if applicable

### CODE REVIEW

**Review by cbartoszDell — DISMISSED** [review]

**Review by HarishH-DELL — APPROVED** [review]
LGTM

**Review by cbartoszDell — DISMISSED** [review]

File: charts/container-storage-modules/Chart.yaml [hunk 1]
```diff
@@ -30,47 +30,47 @@ type: application
...
 
 - name: csm-replication
-  version: 1.7.0
+  version: 1.7.1
   repository: https://dell.github.io/helm-charts
   condition: csm-replication.enabled
 
 - name: karavi-observability
-  version: 1.7.0
+  version: 1.7.1
```
Comment (KerryKovacevic) [inline charts/container-storage-modules/Chart.yaml:73]: We will not be releasing observability for this patch

Comment (AkshaySainiDell) [reply charts/container-storage-modules/Chart.yaml]: Done

**Review by HarishH-DELL — APPROVED** [review]

**Review by boyamurthy — DISMISSED** [review]
LGTM.

**Review by shanmydell — DISMISSED** [review]

**Review by nitesh3108 — DISMISSED** [review]

**Review by nitesh3108 — APPROVED** [review]

**Review by adarsh-dell — APPROVED** [review]
LGTM.

**Pull request merged**